hackathon-submissions/service-now-plugin.md

```diff
@@ -0,0 +1,3 @@
+# Service now mattermost integration
+
+https://github.com/pradeepmurugesan/mattermost-plugin-servicenow
```

Comment:
The links seams broken

Reply:
@hanzei sorry my bad..  made the repo public now.